website/docs/about/contribution.md

```diff
@@ -27,7 +27,7 @@ We support a small number of autonomous but coordinated direct contributions. Th
 
 ### Contribute a new icon in Figma
 
-If you’d like to add a new icon to the [icon library](/icons/library), follow the steps in the [Figma library README](https://www.figma.com/file/MYiw4kiVpunIMMw0sBkE1t/%E2%9C%8F%EF%B8%8F-Flight-Development?node-id=566%3A1129&t=Bbflj3UUaWVyhamn-4).
+If you’d like to add a new icon to the [icon library](/icons/library), follow the steps in the [Figma library README](https://www.figma.com/design/MYiw4kiVpunIMMw0sBkE1t/%E2%9C%8F%EF%B8%8F-HDS-Icons-%5BDevelopment%5D?m=auto&node-id=169-0&t=BhGy7PceZ2LrlRCL-1).
```

Comment:
This link leads to the cover page for the file, should it lead instead to the README frame on the main page?

Reply:
Yep, good call, I'll update this.